include/packetgraph/vtep.h

```diff
@@ -54,10 +55,35 @@ struct ether_addr *pg_vtep_get_mac(struct pg_brick *brick);
  * @param	multicast_ip the multicast ip to associate to the VNI
  * @param	errp an error pointer
  */
-int pg_vtep_add_vni(struct pg_brick *brick,
-		    struct pg_brick *neighbor,
-		    uint32_t vni, uint32_t multicast_ip,
-		    struct pg_error **errp);
+int pg_vtep4_add_vni(struct pg_brick *brick,
+		     struct pg_brick *neighbor,
+		     uint32_t vni, uint32_t multicast_ip,
+		     struct pg_error **errp);
+
+
+/**
+ * Add a VNI to the VTEP
+ *
+ * NOTE: Adding the same VNI twice is not authorized and will result in an
+ *       assertion
```

Comment:
Is this still true ?
Maybe we have to fix this comment or fix this behavior by just setting errp